pkg/cmd/version/version.go

```diff
@@ -15,22 +15,85 @@
 package version
 
 import (
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"time"
 
+	"github.com/blang/semver"
 	"github.com/spf13/cobra"
 )
 
 // NOTE: use go build -ldflags "-X github.com/tektoncd/cli/pkg/cmd/version.clientVersion=$(git describe)"
 var clientVersion = "dev"
 
+const latestReleaseURL = "https://api.github.com/repos/tektoncd/cli/releases/latest"
+
 // Command returns version command
 func Command() *cobra.Command {
 	var cmd = &cobra.Command{
 		Use:   "version",
 		Short: "Prints version information",
 		Run: func(cmd *cobra.Command, args []string) {
 			fmt.Fprintf(cmd.OutOrStdout(), "Client version: %s\n", clientVersion)
+
+			current, latest, err := getVersions()
+			if err != nil {
+				// Suppress the error
```

Comment:
I know this part can be a little bit controversial. 

I want the version command to succeed even if the latest version check fails since, after all, the information about the latest version is not critical. Also, the latest version check can fail relatively easily if, for example, the user does not have an internet access. 

However, I feel ignoring the error is a little bit too extreme, so probably need to print out the error at least with the current version info?（Or if there is a debug flag, I should probably print out the error as a debug info）.

Reply:
Yes, I would vote for printing the error in a warning message on `stderr` or something :+1:

Reply:
Thanks for your comment! Now I fixed to print out the error to the stderr, so please check the commit: https://github.com/tektoncd/cli/pull/134/commits/58ca5b2ec076cf54a19277559eead620463d666d